test/ScanDependencies/module_deps.swift

```diff
@@ -80,6 +80,9 @@ import SubE
 // CHECK-NEXT: {
 // CHECK-NEXT: "swift": "A"
 // CHECK-NEXT: }
+// CHECK-NEXT: {
+// CHECK-NEXT:   "swift": "_cross_import_E"
```

Comment:
I think we should report the YAML files we loaded as dependencies, right?

Reply:
Good catch! Does it suffice if we add the YAML file as part of the sources of this depending module?